build/opusfile-0.7/win32/VS2015/opusfile.vcxproj

```diff
@@ -108,7 +108,7 @@
       <PreprocessorDefinitions>WIN32;_DEBUG;_CRT_SECURE_NO_WARNINGS;_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <MultiProcessorCompilation>true</MultiProcessorCompilation>
       <ProgramDataBaseFileName>$(OutDir)$(TargetName).pdb</ProgramDataBaseFileName>
-      <AdditionalIncludeDirectories>../../include;../../../../include/opus</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>../../include;../../../opus-1.1.2/include</AdditionalIncludeDirectories>
```

Comment:
Yes, I guess I could change that. 
The truth is that pegasus fork already sets the libdir and includedir globally (as an environament variable) so now there is no need for the projects to explicitly point to the generated lib/include.
I dont remember if I needed to change that, or it was simply that i saw it while doing other things.